steering/README.md

```diff
@@ -11,13 +11,13 @@ See [CHARTER.md](CHARTER.md).
 
 &nbsp; | Name | Company | Profile | Seat type
 ---|---|---|---|---
+<img width="30px" src="https://avatars.githubusercontent.com/u/27636385?v=4"> | Srihari Angaluri | IBM | [angaluri](https://github.com/angaluri) | Contribution seat
 <img width="30px" src="https://avatars0.githubusercontent.com/u/132510?s=460&v=4"> | Craig Box | Google | [craigbox](https://github.com/craigbox) | Contribution seat
 <img width="30px" src="https://avatars2.githubusercontent.com/u/1923135?s=400&u=5c04620901ef2b402be16677f1cae3c5d376bea0&v=4"> | Zack Butcher | Tetrate | [ZackButcher](https://github.com/ZackButcher) | Community seat
 <img width="30px" src="https://events.istio.io/istiocon-2021/images/organizers/kevin-conner.jpg"> | Kevin Conner | Red Hat | [knrc](https://github.com/knrc) | Contribution seat
-<img width="30px" src="https://avatars.githubusercontent.com/u/27636385?v=4"> | Srihari Angaluri | IBM | [angaluri](https://github.com/angaluri) | Contribution seat
-<img width="30px" src="https://avatars.githubusercontent.com/u/8482383?v=4"> | April Kyle Nassi | Google | [thisisnotapril](https://github.com/thisisnotapril) | Contribution seat
 <img width="30px" src="https://avatars0.githubusercontent.com/u/1562325?s=400&v=4"> | Sven Mawson | Google | [smawson](https://github.com/smawson) | Contribution seat
 <img width="30px" src="https://avatars.githubusercontent.com/u/17597872?s=460&u=e94e38992c92a7a9e57ae0797ef74799e2e2e201&v=4"> | Jason McGee | IBM | [CloudJason](https://github.com/CloudJason) | Contribution seat
+<img width="30px" src="https://avatars.githubusercontent.com/u/8482383?v=4"> | April Kyle Nassi | Google | [thisisnotapril](https://github.com/thisisnotapril) | Contribution seat
```

Comment:
Just your name, sorry Srihari.  April's surname sorts as "K", not "N".  This happens every time. 😁